arch/x86_64/mcount-dynamic.c

```diff
@@ -15,7 +15,7 @@
 #define XRAY_SECT  "xray_instr_map"
 #define MCOUNTLOC_SECT  "__mcount_loc"
 
-#define CALL_INSN_SIZE  5
+#define INSTRUMENT_CODESIZE  5
```

Comment:
What about aarch64?  It's using `CODE_SIZE`..  Well, I won't object but generally think that pure renaming without a practical reason is not necessary..

Reply:
@namhyung 
I don't want to change codes that I don't wroten. but I think that Instrumentation is perfect fit technical terms for this case. "patch" or "code" describe what itself is. but it is better that variable name describe what is it and how to use it. 

I had not enough skill for considering it when naming it. I apology about it.